src/transformers/training_args.py

```diff
@@ -458,8 +456,8 @@ class TrainingArguments:
             "help": "When resuming training, whether or not to skip the first epochs and batches to get to the same training data."
         },
     )
-    sharded_ddp: ShardedDDPType = field(
-        default="no",
+    sharded_ddp: str = field(
+        default="",
```

Comment:
perhaps list the choices here? and perhaps a very small example of combining 2 of them in the value, since it's not a usual pattern - a user might struggle here.

Reply:
I agree with @stas00!